parser/parser.y

```diff
@@ -657,6 +659,7 @@ import (
 	LoadDataStmt		"Load data statement"
 	LocalOpt		"Local opt"
 	LockTablesStmt		"Lock tables statement"
+	LockClause         "Alter table lock clause"
```

Comment:
I don't know how to keep alignment in vscode or sublime text. I have to do it in vim.